docs/semgrep-cloud-platform/getting-started.md

```diff
@@ -94,6 +76,82 @@ Using Semgrep Supply Chain with Semgrep Cloud Platform provides users with the V
+
+If you are a **team member joining an existing organization**, see [Joining an organization](#joining-an-organization).
+
+If you are a **personal user**, or simply want to run scans, see [Running scans](#running-scans).
+
+## Adding an organization
+
+This process creates an **organization account**, which your team members in GitHub or GitLab can join. Organization accounts share custom rules that are written by fellow team members, as well as records of findings and scans. 
+
+1. Sign in to Semgrep Cloud Platform. A dialog box appears.
+2. Click **Create an organization**.
+![DESCRIPTION](/img/create-an-org.png#md-width)
+3. Enter your **Organization display name**. This name typically corresponds to the name of your GitHub or GitLab org.
+4. Click **Create**.
+5. Your organization is created and you are prompted to start a scan. Instead of starting a scan, click **<i class="fa-solid fa-gear"></i> Settings** in the navigation menu.
+6. Click **Source code managers**.
+7. Select the source code manager that contains the org to connect to Semgrep Cloud Platform.
+![DESCRIPTION](/img/source-code-manager.png#md-width)
+8. Follow the steps to connect your source code manager. You can review permissions needed by Semgrep in [Requested permissions for GitHub and GitLab](#requested-permissions-for-github-and-gitlab).
+
+## Joining an organization
+
+:::info Prerequisites
+- Your admin or inviter should have an existing organization (org) in Semgrep that is connected to the org in either GitHub or GitLab.
+- You must be a member of the GitHub or GitLab org that is connected to Semgrep.
+:::
+
+For **members to join an organization**, perform the following steps to be added to the org:
+
+1. Sign in to Semgrep Cloud Platform. A **Welcome to Semgrep** dialog box appears.
+2. Check that the organization under the **Join an organization** button is the correct org to join. If there is more than 1 org, select the org to join.
+![DESCRIPTION](/img/join-an-org.png#md-width)
+3. Click **Join an organization**.
+
+## Running scans
+
+### Starting a SAST and SCA scan on a remote repository
+
+<PlatformAddRepo />
+
+#### Detecting GitHub repositories
+
+<PlatformDetectGhRepos />
+
+:::tip Product-specific information
+* To learn more about SAST scans on your codebase, see [Getting started with Semgrep Code](/semgrep-code/getting-started).
+* To learn more about SCA scans for your third-party dependencies, see [Getting started with Semgrep Supply Chain](/semgrep-supply-chain/getting-started).
+* Both products are **free for up to 10 contributors**. See [Usage limits](/usage-limits) to learn more about contributors and usage limits.
+:::
+
+### Starting a local repository scan and sending findings to SCP
+
+You can send scan results from a local repository to Semgrep Cloud Platform. The local repository is a separate **Project** from its remote counterpart so the findings records do not overwrite each other and are kept separate.

```

Comment:
I think this is usually true, but is it always true?

Reply:
There is one case I haven't verified, which is a case when someone pulls an org repo (say semgrep) then uploads findings from semgrep to the org - what happens when another user does that? I'll ask the cloud folks.

Reply:
I think it might overwrite in that case, potentially. My observation is it's usually `org_name/repo_name` from CI and just `repo_name` locally. 

I was also thinking about if someone goes and reads about environment variables and sets the name, but that's pretty unlikely when they're just getting started.

Reply:
Fixed in latest, I'll add a screenshot in the main comment body.